### PR TITLE
Feat/comment endpoint test

### PR DESCRIPTION
**Pull request by fahmirzldi**
Add test for comment controller, model, service

### CODE REVIEW

File: app/main/model/comment.py [hunk 1]
```diff
@@ -59,12 +55,17 @@ def create_comment(self, data):
...
+        return [comment.serialize() for comment in comments]
 
     def get_comment_by_id(self, public_id):
-        return self.query.filter_by(public_id=public_id, visible=True).first()
-
+        return self.query.filter_by(public_id=public_id, visible=True).first().serialize()
+    
     def delete_comment(self, public_id):
-        comment = self.get_comment_by_id(public_id)
+        # comment = self.get_comment_by_id(public_id)
```
Comment (fatjan) [inline app/main/model/comment.py:67]: should we remove this commented line?

File: app/main/model/comment.py [hunk 2]
```diff
@@ -74,7 +75,8 @@ def delete_comment(self, public_id):
             return comment.serialize()
 
     def update_comment(self, public_id, data):
-        comment = self.get_comment_by_id(public_id)
+        # comment = self.get_comment_by_id(public_id)
```
Comment (fatjan) [inline app/main/model/comment.py:78]: this one too

File: app/main/service/comment_service.py [hunk 3]
```diff
@@ -28,7 +28,7 @@ def get_all_comments():
 def get_a_comment(public_id):
     try:
         comment = comment_model.get_comment_by_id(public_id)
-        comment.serialize()
+        # comment.serialize()
```
Comment (fatjan) [inline app/main/service/comment_service.py:31]: just remove the line if it's not used

**Comment by fahmirzldi** [comment]
all unnecessary comment deleted

File: app/main/model/comment.py [hunk 4]
```diff
@@ -74,7 +74,7 @@ def delete_comment(self, public_id):
             return comment.serialize()
 
     def update_comment(self, public_id, data):
-        comment = self.get_comment_by_id(public_id)
+        comment = self.query.filter_by(public_id=public_id, visible=True).first()
         if not comment:
             return None
```
Comment (AchmadQiz) [inline app/main/model/comment.py:79]: can you implement try catch block and add related error message? can refer to the methods in the user model

Comment (fahmirzldi) [reply app/main/model/comment.py]: kindly check for updated try catch

**Review by fatjan — CHANGES_REQUESTED** [review]
Please update as per suggestions. Thank you

File: app/main/model/comment.py [hunk 5]
```diff
@@ -40,67 +40,88 @@ def save(self):
...
-        return comment.serialize()
-
-    def update_visibility(self, public_id, visible=True):
-        comment = self.query.filter_by(public_id=public_id).first()
-        if not comment:
-            return None
-        else:
-            comment.visible = visible
+        try:
+            # comment = self.get_comment_by_id(public_id)
```
[the remark this answers is not in the document]
Comment (fahmirzldi) [reply app/main/model/comment.py]: Done

**Comment by fahmirzldi** [comment]
Updated as suggested

**Review by fatjan — APPROVED** [review]

**Pull request merged**